### PR TITLE
reset output height to previous hight instead of hardcoded value

### PR DESCRIPTION
**Pull request by grosser**
- when scrolling around output got resized to 550px
 - move following logic into a sane order and name the ids after the button labels
 - do not scroll the user when in expanded mode
 - do not follow/unfollow when in expanded mode

### CODE REVIEW

File: app/assets/javascripts/deploys.js [hunk 1]
```diff
@@ -126,43 +129,47 @@ $(function () {
...
-    }
-  });
+  $("#output-no-follow").click(function() {
+    following = false;
 
-  $("#output-grow").click(function() {
-    growOutput();
+    shrinkOutput();
 
     $("#output-options > button").removeClass("active");
```
Comment (sandlerr) [inline app/assets/javascripts/deploys.js]: this is a bug right? select "Don't Follow" shouldn't deactivate "Expand"?

Comment (sandlerr) [inline app/assets/javascripts/deploys.js]: or if they're mutually exclusive then we should have this line in the other handler too?

**Comment by grosser** [comment]
it's a 3-way modal "Follow", "Don't Follow", "Expand"

File: app/assets/javascripts/deploys.js (hunk 1 above)
Comment (grosser) [inline app/assets/javascripts/deploys.js]: the other 3 already have this line, but it was messy ... refactored into a single 'activate' method

**Comment by grosser** [comment]
forgot to push ... updated now

**Comment by grosser** [comment]
I like that

**Comment by grosser** [comment]
... this needs more rework ... merging for now ... can start a new PR from that

**Pull request merged**